README.md

```diff
@@ -70,6 +70,13 @@ At this moment you should be enjoying a fully featured augmented reality view sh
 
 And that's all!! For more information on how to add custom overlay views, configure the radar position, start the engine in different orientations or better understanding of the front/floor looking feature, please run the provided example project.
 
+## For iOS 8 Support
+Add these key in Info.plist of your app
+NSLocationAlwaysUsageDescription
```

Comment:
Maybe add some more info on what the key means, and which value is expected. Or a link to the documentation?